napari/components/dims.py

```diff
@@ -225,6 +231,53 @@ def set_range(
                     ax = assert_axis_in_bounds(int(ax), self.ndim)
                     full_range[ax] = r
                 self.range = full_range
+        if restore_point is not None:
+            new_point = self._get_point_in_ndim(restore_point)
+            self.set_point(range(self.ndim), new_point)
+
+    @staticmethod
+    def rounded_division(min_val, max_val, precision):
```

Comment:
Would be great to not add a public duplicate of `ViewerModel.rounded_division`. Either hide/privatize this with a leading underscore, or find a better way to share this declaration and implementation.

Reply:
i moved it to `napari.utils.misc`